document/src/tests/documenttestcase.cpp

```diff
@@ -1251,4 +1265,42 @@ TEST(DocumentTest, testDeserializeMultiple)
     EXPECT_EQ(correct, sv3);
 }
 
+TEST(DocumentRejectTest, requireThatFixedSizeFieldValuesAreDetected) {
```

Comment:
Consider putting these tests in a separate test file. documenttestcase.cpp is already pretty bloated.

Reply:
Fixed